src/applications/vaos/actions/appointments.js

```diff
@@ -320,20 +321,23 @@ export function confirmCancelAppointment() {
           appointmentRequestDetailCode: ['DETCODE8'],
         });
       } else {
+        const facilityId = getVARFacilityId(appointment).replace('var', '');
+
         const cancelData = {
-          appointmentTime: appointment.appointmentDate.format(
+          appointmentTime: moment(appointment.start).format(
             'MM/DD/YYYY HH:mm:ss',
           ),
-          clinicId: appointment.clinicId,
-          facilityId: appointment.facilityId,
+          clinicId: getVARClinicId(appointment).replace('var', ''),
```

Comment:
I think this would include `var` in the id? I think you could actually remove the `var` prefix in getVARClinicId. We won't have a direct equivalent in FHIR to the clinicId (since HealthcareServices are site id, clinic id pairs).